sos/report/plugins/fail2ban.py

```diff
@@ -0,0 +1,36 @@
+# This file is part of the sos project: https://github.com/sosreport/sos
+#
+# This copyrighted material is made available to anyone wishing to use,
+# modify, copy, or redistribute it subject to the terms and conditions of
+# version 2 of the GNU General Public License.
+#
+# See the LICENSE file in the source distribution for further information.
+
+from sos.report.plugins import Plugin, IndependentPlugin
+
+
+class Fail2Ban(Plugin, IndependentPlugin):
+
+    short_desc = "Fail2Ban daemon"
+
+    plugin_name = "fail2ban"
+    profiles = ('security',)
+    packages = ('fail2ban', 'fail2ban-server')
+    servicess = ('fail2ban',)
+
+    common_dir = '/etc/fail2ban'
+
+    def setup(self):
+
+        self.add_copy_spec([
+            f'{self.common_dir}/fail2ban.*',
+            f'{self.common_dir}/jail.*',
+            f'{self.common_dir}/filter.d',
+            f'{self.common_dir}/action.d',
+            f'{self.common_dir}/jail.d',
```

Comment:
that's already been removed; I think you've commented on an older commit :)

Reply:
Eh, blame github WebUI :). I reviewed current code changes, clicked to review changes and Submit review (as usual). Since I had some unresolved comment on an outdated code change, WebUI repeated my comment, apparently..? /o\

ACK to the current PR, just to be sure :)